src/Extensions/Logging/ServiceInitializationEventSource.cs

```diff
@@ -5,13 +5,13 @@
 using System.Diagnostics.Tracing;
 using Microsoft.Omex.Extensions.Abstractions.EventSources;
 
-namespace Microsoft.Omex.Extensions.Hosting.Services
+namespace Microsoft.Omex.Extensions.Logging
 {
 	/// <summary>
 	/// Service Fabric event source
 	/// </summary>
 	[EventSource(Name = "Microsoft-OMEX-HostLogs")] //TODO: new event source should be registred GitHub Issue #187
-	internal sealed class ServiceInitializationEventSource : EventSource
+	public sealed class ServiceInitializationEventSource : EventSource
```

Comment:
I don't think it needs to be public

Reply:
This needs to be public so that this package and hosting can use it

Reply:
Could we use InitializationLogger in Hosting instead? It looks like it's proper usage